jobs/configure-eirini-bosh/templates/eirini-daemonset.yaml.erb

```diff
@@ -14,7 +14,7 @@ spec:
       labels:
         name: eirini-loggregator-fluentd
     spec:
-      serviceAccountName: "<%= p('eirini.service_account') %>"
+      serviceAccountName: pas-system-service-account
```

Comment:
We should probably not default to this Pivotal-specific value

Reply:
@jspawar @madamkiwi Agreed, I think it should have been `"<%= p('opi.system_namespace') %>-service-account"`; might have just missed it.

Reply:
Good point @jspawar . @jrussett and I have changed this.